host-interaction/process/inject/use-thread-execution-hijacking.yml

```diff
@@ -0,0 +1,25 @@
+rule:
+  meta:
+    name: use thread execution hijacking
+    namespace: host-interaction/process/inject
+    author: 0x534a@mailbox.org
+    scope: function
+    att&ck:
+      - Defense Evasion::Process Injection::Thread Execution Hijacking [T1055.003]
+    examples:
+      - 787cbc8a6d1bc58ea169e51e1ad029a637f22560660cc129ab8a099a745bd50e:0x4044C7
+  features:
+    - and:
+      - optional:
+        - or:
+          - api: kernel32.OpenThread
+          - api: ntdll.NtOpenThread
+          - api: ntdll.ZwOpenThread
```

Comment:
looks like another good (library) rule candidate `open thread`

Reply:
Fixed in aa94a33